nexus/src/app/rack.rs

```diff
@@ -608,15 +609,30 @@ impl super::Nexus {
                 .bgp_peers
                 .insert("phy0".to_string(), BgpPeerConfig { peers });
 
-            let link = LinkConfigCreate {
-                mtu: 1500, //TODO https://github.com/oxidecomputer/omicron/issues/2274
-                lldp: LldpServiceConfigCreate {
+            let lldp = match &uplink_config.lldp {
+                None => LldpLinkConfigCreate {
                     enabled: false,
-                    lldp_config: None,
+                    ..Default::default()
+                },
+                Some(l) => LldpLinkConfigCreate {
+                    enabled: l.status == LldpAdminStatus::Enabled,
```

Comment:
Is this translation problematic? Seems weird to me to report "enabled: false" when we might be still be allowing rx_only or tx_only traffic.

(This isn't my domain of expertise, so I trust your judgement, just seems odd to have those settings convert to "LLDP if off!")

Reply:
There is a mismatch between what we support, and what the LLDP RFC supports.  The protocol allows a node to be rx-only or tx-only, but the old LLDP tables only included enabled/disabled.  It looks like Cisco at least does support all 4 options, so maybe we should as well.  I'll run this by Ry and/or Trey to see whether we should offer that flexibility.